include/crm/common/output.h

```diff
@@ -57,18 +57,18 @@ typedef enum {
  * \brief Further modify the output of sections
  */
 typedef enum {
-    pcmk_show_brief         = 1 << 0,
-    pcmk_show_clone_detail  = 1 << 1,
-    pcmk_show_node_id       = 1 << 2,
-    pcmk_show_implicit_rscs = 1 << 3,
-    pcmk_show_timing        = 1 << 4,
-    pcmk_show_inactive_rscs = 1 << 5,
-    pcmk_show_rscs_by_node  = 1 << 6,
-    pcmk_show_pending       = 1 << 7,
-    pcmk_show_rsc_only      = 1 << 8,
-    pcmk_show_failed_detail = 1 << 9,
-    pcmk_show_feature_set   = 1 << 10,
-    pcmk_show_description   = 1 << 11,
+    pcmk_show_brief         = UINT32_C(1) << 0,
```

Comment:
Technically this changes public API (in a way that should not affect behavior on reasonable systems)